src/inertiajs-stubs/providers/InertiaJsServiceProvider.stub

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Session;
+use Illuminate\Support\ServiceProvider;
+use Inertia\Inertia;
+
+class InertiaJsServiceProvider extends ServiceProvider
+{
+    public function register()
+    {
+        $this->registerInertia();
```

Comment:
Any reason to not just inline this into the `register()` function? Not sure I see the benefit of the extra method here.

Reply:
No specific reason; removed!